src/backend/joanie/settings.py

```diff
@@ -357,15 +359,17 @@ class Test(Base):
             "API_TOKEN": "FakeEdXAPIKey",
             "BACKEND": "joanie.lms_handler.backends.dummy.DummyLMSBackend",
             "BASE_URL": "http://localhost:8073",
-            "SELECTOR_REGEX": r"^.*/courses/(?P<course_id>.*)/course/?",
-            "COURSE_REGEX": r"^.*/courses/(?P<course_id>.*)/course/?$",
+            "SELECTOR_REGEX": r"^(?P<course_id>.*)$",
+            "COURSE_REGEX": r"^(?P<course_id>.*)$",
```

Comment:
I didn't understand this change?

Reply:
Because, by default the `resource_link` generated by `CourseRunFactory` is a simple url so I just update the regex here to be able to use the LMS Dummy Backend without to have to set the resource_link manually in each test where it is necessary to use lms backend methods